spectrum-cardano-connector/src/lib.rs

```diff
@@ -0,0 +1,320 @@
+use crate::rocksdb::{serialize_tx, Block, ChainCacheRocksDB, RocksConfig};
+use pallas_network::{
+    facades::PeerClient,
+    miniprotocols::{
+        chainsync::{self, NextResponse},
+        Point,
+    },
+};
+use pallas_traverse::{MultiEraBlock, MultiEraHeader};
+use spectrum_chain_connector::{DataBridge, DataBridgeComponents, TxEvent};
+
+mod rocksdb;
+
+pub struct CardanoDataBridge {
+    pub receiver: tokio::sync::mpsc::Receiver<TxEvent<Vec<u8>>>,
+    tx_start: tokio::sync::oneshot::Sender<()>,
+}
+
+impl DataBridge for CardanoDataBridge {
+    type TxType = Vec<u8>;
```

Comment:
Have to deal with serialized bytes of the transactions due to Pallas' use of copy-on-write (Cow) primitives. See: https://github.com/txpipe/pallas/blob/ea9c4e8b59fe5cbaaf73c649b44347a066f70960/pallas-traverse/src/lib.rs#L78